### ISSUE TITLE
Sugestion

### ISSUE THREAD

**Issue by Darktotaled** [TeamBR-Modding/NeoTech#100]
I see 4 tiers in neotech
Tier 0: Grinder
Tier 1: Furnace generator, electric furnace and electric crusher
Tier 2: Integration with pipes
Tier 3: Upgrades

Could you guys take out sticky piston from the pipe recipe and exchange it for something less rare to find? The more things you do with neotech, the more you need pipe interfaces and it's really boring to hunt through the map to use a whole tier in the mod.

I saw the color changing energy bar on the machines. The idea is cool but it looks a little bit unpolished. I guess i'm too confortable with the bright red from Redstone Flux.

The control upgrade to be able to mess with inputs and outputs in machines just pushed automation farther. Guess you can use the control upgrade to add filters to machines and let the output and input to be already there.

About the energy cells, going to the nether to be able to make a basic energy cell is kinda much work. we have 3 good materials to conduct eletricity: Tin, copper and gold. What about using a recipe with them in the place of the comparators?

As a player that usually plays with Thermal expansion, buildcraft, railcraft, mekanism, ender I/O and others, I really like your mod and want to see it among the bigger ones. It looks like I'm just being a casual child complaining about progression but i think that the mod need a little bit of balancing at the basic level of machinery.

Thanks btw
awesome mod and keep the good work :D

**Comment by pauljoda** [comment]
We have been thinking about balance. I personally rarely actually play the game so sometimes its difficult for me to judge. I'm also terrible at recipes. I like your ideas about electric components instead of comparators. Not sure why we chose those. 

The pipe recipe has been change. We just flat removed the sticky piston. Should make things much easier. I agree I hated the sticky pistons after using them in world. 

The color changing was something I added because I thought a plain red bar was too plain. I'm not sure if I like it starting from black but I feel it adds a little more to the mod

I like your idea of the filters in machines, I may add that however I will limit it to 3 at most 4 items to filter with no advanced matching like NBT etc. This is just because that is handled with pipes

Feel free to keep the ideas coming. I'll leave this open for you and others to chime in. We need experienced players to help us balance it to the standard

**Comment by BrewingCoder** [comment]
the energy cell quartz thing is a little off for the first tier cell as far as balance goes I agree.

I'd also like to see the pipes have an opaque option that would bypass animation.  Something that you don't see out there is the ability to modify the pipe after it's been placed.  You always want the animated pipes when first setting up your gear because it's really required when debugging automation.  But once the puppy is set you want to avoid lag and cover them.  would always be nice to make the pipes opaque "after" going through the build.

**Comment by Dyonovan** [comment]
One of the things we are planning to add is a cover system for the pipes that should do what you want it to do. If the cover is in place we could turn off the animations

**Comment by pauljoda** [comment]
I can add a toggle to the interface pipes to hide things that they have extracted as well. Isn't hard to add

**Comment by Dyonovan** [comment]
09519e02be75fa17effc4548612a96c05827b3e9

Changed Basic RF Storage to not require a trip to the nether

**Comment by Darktotaled** [comment]
about the pipe animation, what about a wool over wool recipe to create an overlay over the pipe. It could be colored and would attach to any segment of pipes either interface or basic pipe

**Comment by Darktotaled** [comment]
As i see pipe automation as a tier 2 thing, you could use pipe hopper redstone in the recipe where pipe is the structure, redstone the control and hopper the extraction ability of the pipe, so the recipe has a sense. I must say I really appreciate you guys. You are really hardworking people and i love discussing ideas :D

**Comment by Darktotaled** [comment]
As an industrial/tech mod i guess you guys should have a weapon of mass destruction of terrain also known as a drill/quarry/miner/excavator thing. I was thinking about something that no one tried yet. A multiblock platform with a dril spinning in the middle pointing down. The platform would be a 5x5 with some columns near the middle and the drill would be a 3x3x3. You could install hard drive upgrades to increase area of excavation(it would be 3x3 area below the drill by default), processor to increase speed, control for redstone and filter purposes and expansion for silk touch ability( spends more energy btw). It would tear down any block below it and put the blocks in it 9 slot storage that could be acessed by any part of the platform. I'm still learning Java and i suck at drawing so the best i can do for you guys is giving the ideas i have for someone capable of making them true.

while i was typing this wall of text i decided to draw a little bit and the result sucked really hard, but i guess it can show you what i tried to explain.
http://prntscr.com/a25omf

I hope you like it o/

**Comment by pauljoda** [comment]
That is on our list. I'm not sure if we will be going the multiblock route
for the moment. The idea is good and I agree it would be cool. That being
said, I have no idea what we will do for a quarry. I can't promise I will
use this idea, but ill keep it in mind as I work out the mechanics. Feel
free to keep the ideas coming

**Comment by Darktotaled** [comment]
I'm glad to be helping :D Any thoughts on a techsuit or upgradable tools?

**Comment by Darktotaled** [comment]
what is the main purpose of neotech? I wanna know so i can have a guideline for my ideas

**Comment by Dyonovan** [comment]
Neotech was initially created as there were no tech mods for 1.8 (which there really isnt a lot now either)

Basically its a tech mod. 

Upgradeable tools are on the list (using RF)

**Comment by Darktotaled** [comment]
What about using more Neo on the mod adding elevators, hydro generators, windmills, biomass, gravity manipulator(for flying), sun based machines(like a growth accelerator for plants), a greenhouse structure, an auto harvesting/planting/storing machine, a metal boat for lava rides in the nether, an energy gun, modifications to be installed in a techsuit, a techsuit, a cobblegen(really needed), a craftable type of fuel(liquid or solid) and a in-game book about the mod(would be really really awesome btw).

**Comment by Dyonovan** [comment]
O god.....

Love the ideas. For growth accelerator, look at the Miniature sun.

**Comment by Darktotaled** [comment]
There is so many things Neotech can achieve and make :D the Ideas just keep coming. The dimensional storage was an amazing idea btw. Started using it yesterday along storaged drawers

**Comment by Darktotaled** [comment]
there is something i was curious about for some time... What BR stands for on TeamBR?

**Comment by pauljoda** [comment]
Beyond Reality. We are a group of people from that team but focused on mods and not mod packs. http://www.beyondrealitypack.com/

**Comment by Darktotaled** [comment]
I saw bevo playing that modpack _----_

**Comment by Darktotaled** [comment]
Is dimensional crates upgrades on the list too?

**Comment by Dyonovan** [comment]
For sure!

Spent today implementing a Void Tank and a Mob Catcher.

https://gyazo.com/97e289ea4853f043e98f70955072362c
https://gyazo.com/9d50deb2e8310da44962763c232993ed

**Comment by Darktotaled** [comment]
looks amazing. _---_ Keep the good work guys. You rock

**Comment by Darktotaled** [comment]
Can you make a trash can too? the thrash bag is good to carry through out the world but it would be nice to pipe things we dont want into a void chest

**Comment by Darktotaled** [comment]
do you know what would look really nice? a multicolored energy bar. Starting at red, goes through yellow and finishes at green. You could make "cuts" in the bar dividing it in 10 parts so we would know the percentage of stored energy.

**Comment by pauljoda** [comment]
The flushable chest will void items when given a Redstone signal

**Comment by Darktotaled** [comment]
I had a really neat idea :D What about teleporting a entire building? You could place markers or Time-Space manipulators(8 of them: 4 in the base level, 4 in the topmost level) in the edges of the house or factory or statue or anything else. Them you put the Time-Space teleporter next to one of the manipulators, them you proceed to mark the area you want the building, add power to the teleporter, redstone signal and puff. It's there

**Comment by Darktotaled** [comment]
Lol, I really forgot about the flushable chest @pauljoda but i was thinking about a block without the need of interaction just like the void tank

**Comment by Darktotaled** [comment]
Started playing with solar panels and... they all have the same energy buffer? '-' Basic solar panel> rf storage cell+ glass+ redstone = 32000 rf... ok you took a cell that holds 25k rf and made a solar panel with an internal buffer of 32k, it's an improvement, it's good, but advanced solar panel> advanced rf storage+ 8 basic solar panels = 32k energy buffer? shouldn't it be at least 8 times the basic one cause we have 8 of them in the recipe, even so the advanced cell would be ignored right? It's a 1kk rf storage unit.

As it is now, it's better making 8 basic ones than a advanced one cause the energy output would be the same but we would have 8 times the energy buffer.

Keep Rockin' guys. You are the best o/

**Comment by Dyonovan** [comment]
Dang it.. No one was supposed to notice that :P Its on my list of things to do today

**Comment by Darktotaled** [comment]
I'm trying everything in the mod and trying to integrate it with other mods so i can see what it can do. Everything I see, I open the browser and start talking/typing.

**Comment by Darktotaled** [comment]
Like this:   http://prntscr.com/a2ujc7

Power advantage doesn't connect to thaumcraft, so i'm using your fluid pipe to do the integration and i love it. It's a simple pipe and really useful in a lot of things

**Comment by pauljoda** [comment]
Great. Love the feedback keep it coming

**Comment by Darktotaled** [comment]
Does the tanks auto output at the bottom? Put one of the diamond ones over a Buildcraft fluid pipe and it started outputting

**Comment by Darktotaled** [comment]
How many items a dimensional crate can hold? 64 stacks?

**Comment by pauljoda** [comment]
Yes, tanks drain down. You can also stack them and if the tank is full, it will try to fill those above it, like Buildcraft tanks. The crate holds 64 I believe. We have plans to make it bigger with upgrades

**Comment by Darktotaled** [comment]
seems nice :D thanks

**Comment by Darktotaled** [comment]
With buildcraft pipes they output the liquid no matter the type of pipe below them. Was getting some oil and it took really long till i noticed that the oil was going up and down the tanks

**Comment by Darktotaled** [comment]
With buildcraft pipes they output the liquid no matter the type of pipe below them. Was getting some oil and it took really long till i noticed that the oil was going up and down the tanks

**Comment by Darktotaled** [comment]
I've been checking this github more often than my facebook '-' Been thinking about another fuel source. What about a blend of gold dust, redstone dust and coal dust? Energetic dust that can be smelt into energetic crystal and burned on furnaces or furnaces generator for 8 times normal coal output.

**Comment by Darktotaled** [comment]
Would be nice to see how much RF is the thermal binder eating. Like the furnace generator that shows how much it produces and changes when you install upgrades, i'd like to see how much thermal binder drains with and without upgrades so i can do my math.

**Comment by pauljoda** [comment]
I plan to add current state info to the top of the description tab

**Comment by Dyonovan** [comment]
@Darktotaled Added different energy storage for different tiers of solar panels.

6a609d95944d8f0c9dc83a03e9be0d0cc5f6c9cc

**Comment by Darktotaled** [comment]
Yay @Dyonovan you are the best ;) I'm using neotech with a friend and we are almost at the end of neotech. waiting for more content :D

**Comment by Darktotaled** [comment]
Was thinking about the cover system. What about mixing sand, clay and a water bucket to make cement and we could dye it before throwing at the furnace. You could right click a pipe with the cement in hands and it would attach to it transform that section in a block

**Comment by Darktotaled** [comment]
Fuel: Ethanol from sugar cane
biomass from saplings, furuits, vegetables.
Composite explosive for combustion engines ( gunpowder+ coal powder)
you can use fire and a heating generator also. The generator's heat meter goes up as it senses the nearby temperature(alteerd by lava or netherack on fire) and produces energy.
Steam
Coal powder+ water bucket= liquid fuel(basic one at least)
a mob generator taht uses mob drops as fuel (rarer = more ticks)

**Comment by Darktotaled** [comment]
What about working with alloys? We have now iron, gold, tin and copper. Let's get them mixing o/

**Comment by ph1x3r** [comment]
Perhaps some way of transmitting/receiving items/energy/fluids across distances and dimensions.

a) Basic tx/rx unit that only works for a limited number of chunks (currently loaded chunks perhaps)
b) Advanced tx/rx unit that works anywhere in one dimension.
c) Elite unit that works across dimensions.

The current pipe upgrades would still be appropriate to limit the volume/rate of transfer etc

I am using your electric pump and a fluid generator to fill tanks with lava, but I still need to keep going and fetching the full tanks and dropping off the empties.

**Comment by Darktotaled** [comment]
@ph1x3r Like thermal expansion's Tesseract thing?

**Comment by Darktotaled** [comment]
The upgrade cost from advanced storage to elite storage is too cheap. Change the advanced one to diamonds in the place of the gold ingots and change the recipe of the elite to quartz blocks, nether bricks and stuff. The gap between the basic and advanced is acceptable but advanced to elite is just a tumble. Didn't use the advanced cell but for recipe use.

**Comment by pauljoda** [comment]
Going forward with recipes, if you have suggestions it would be super helpful to use if you could create it in a crafting table and post an image of that here. Just so we have a visual. We need help balancing the recipes so feel free to post ideas

**Comment by Darktotaled** [comment]
http://prntscr.com/a3upak

Bronze ingots, quartz blocks, block of diamond, block of redstone and advanced rf storage.

A little bit more expensive and it claims for an even bigger storage after that.
I really want you guys to explore alloying more and perhaps add an alloying machine cause throwing ingots in a crusher to get the powder when the crusher is connected to a furnace is a pain. Having to disconnect the crusher to get the powders is really messy sometimes.

If you want a list of alloys and other metals i know of just ask me.

Keep rockin' guys

**Comment by pauljoda** [comment]
We have considered alloys and plan to implement something in the near future. Feel free to give your ideas

**Comment by Darktotaled** [comment]
Other metals: Titanium(very good metal for armor and machines)
Aluminum(good mid-point as a energy transfer metal)
Nickel(alloying ingredient)
Zinc(alloying ingredient)

Alloys: Iron+Nickel= Invar(stronger than iron, good for heavy duty machines)
Copper+Zinc= Brass(good at retaining heat)
Iron+coal= Steel(good for armor and machine parts)
Gold+Redstone= Conductant Ingot(awesome at power transmiting)

**Comment by Darktotaled** [comment]
Was thinking about the pipe system. It's really cheap '-' the only thing you put effort into are their upgrades and then i sat a little and started thinking. If the normal structure pipes can connect to other normally and connect to any interface, would a single segment of structure pipe be able to transmit more than one type of output(power and/or items and/or fluid)?

**Comment by pauljoda** [comment]
Yes, the structure pipes can send any amount of any resource. Interface pipes can also be considered part of the network. As long as there are connections, there is communication between the pipes

**Comment by Darktotaled** [comment]
so i can use segments of structure pipes to transfer anything like energy and crushed ores from the crusher to the furnace? 2 interfaces in both machines but only one segment of structure pipe

**Comment by Darktotaled** [comment]
It's 2:25 am and i'm testing NeoTech for bugs

**Comment by Darktotaled** [comment]
http://prntscr.com/a3uxqc

Everything is working fine in my little mess 

Oil Fluid tanks to refinery> refinery to fuel fluid tank> fluid fuel tank to fuel fluid buffer> fuel fluid buffer to combustion engines. Nothing getting wasted now(only space cause i dont care about placement :v)

**Comment by Darktotaled** [comment]
Wouldn't it be nice tiered machines? Like more crushers with different base stats, more secondary output slots, crushing more than one ore at a time. I can help with the recipe if you guys need. I'll be thinking about it. I'll be back in 10 hours. See ya guys

**Comment by Darktotaled** [comment]
Saw @pauljoda talking about batteries. Will it be also tiered? will machines have a slot for batteries so we can make them work for limited time using batteries? I'm already on the hype

**Comment by ph1x3r** [comment]
Darktotaled commented a day ago
@ph1x3r Like thermal expansion's Tesseract thing?

In principle, yes. But I find tesseracts overpowered since they have unlimited throughput and no cost for using them.

**Comment by Darktotaled** [comment]
What about a teleportation chamber then? you fit inside what you want and press a button to send it. Needs rf to transfer and manual activation(can use a redstone clock for automation)

**Comment by mille449** [comment]
I know something tesseract-ish is planned, but it's not simple to implement so it might be a bit still.

**Comment by pauljoda** [comment]
Yes it will be a bit. Things such as that are not easy. However, I have designed a system that should handle it well I just haven't started coding it

**Comment by Dyonovan** [comment]
@Darktotaled Could you do me a REALLY big favor. Can you go through this list and just condense all the suggestions into a new Issue, then close this one. Its getting big and hard to follow at this point. If not, thats ok, I will try to do it.

**Comment by Darktotaled** [comment]
I'll try @Dyonovan
